### ISSUE TITLE
About some warnings 

### ISSUE THREAD

**Issue by suzhenghang** [zhanghang1989/RegNet-Search-PyTorch#12]
I got the fllowing logs after running: python test_flops.py --config-file configs/RegNetX-4.0GF.ini，i wonder whether these warnings are normal?

[INFO] Register count_convNd() for <class 'torch.nn.modules.conv.Conv2d'>.
[INFO] Register count_bn() for <class 'torch.nn.modules.batchnorm.BatchNorm2d'>.
[INFO] Register zero_ops() for <class 'torch.nn.modules.activation.ReLU'>.
**[WARN] Cannot find rule for <class 'arch.regnet.ConvBnAct'>. Treat it as zero Macs and zero Params.
[WARN] Cannot find rule for <class 'arch.regnet.Bottleneck'>. Treat it as zero Macs and zero Params.
[WARN] Cannot find rule for <class 'torch.nn.modules.container.Sequential'>. Treat it as zero Macs and zero Params.
[WARN] Cannot find rule for <class 'arch.regnet.Stage'>. Treat it as zero Macs and zero Params.
[WARN] Cannot find rule for <class 'arch.regnet.GlobalAvgPool2d'>. Treat it as zero Macs and zero Params.**
[INFO] Register count_linear() for <class 'torch.nn.modules.linear.Linear'>.
[WARN] Cannot find rule for <class 'arch.regnet.RegNet'>. Treat it as zero Macs and zero Params.
macs: 3.989G, params: 22.118M

**Comment by HardyC-1021** [comment]
I see the problem too. Maybe it's normal. It doesnot matter
